reviews/views.py

```diff
@@ -77,6 +77,83 @@ def get_context_data(self, **kwargs):
         context['data_url'] = reverse("reviews:api:open", args=[self.committee])
 
         return context
+    
+
+class CommitteeMembersWorkloadView(GroupRequiredMixin, CommitteeMixin, generic.TemplateView):
+    #TODO: Look into improving the should_end_date variable. See notes
+
+    template_name = 'reviews/committee_members_workload.html'
+    group_required = [settings.GROUP_SECRETARY]
+    
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context['decisions'] = self.get_all_open_decisions()
+        context['today'] = date.today()
+        context['counts_dict'] = self.get_review_counts_last_year
+
+        return context
+
+    def get_all_open_decisions(self):
+        '''Returns a queryset with all open decisions'''
+
+        objects = Decision.objects.filter(
+            # This fetches all Decisions which are not approved or need revision
+            go__exact = '',
+            review__proposal__reviewing_committee = self.committee,
+            ).select_related(
+            'reviewer',
+            'review',
+            'review__proposal',
+            ).order_by(
+            "reviewer",
+            "review__date_start"
+            )
+
+        return objects
+    
+    def get_review_counts_last_year(self):
+        '''This method creates a dictionary with an overview of all reviews of
+        the past year for each reviewer, by trajectory type (short route,
+        long route, or revision).'''
+
+        dict={}
+        one_year_ago = date.today() - timedelta(days=365)
+
+        decisions_last_year = Decision.objects.filter(
+        review__date_start__gt = one_year_ago,
+        ).select_related(
+        'review',
+        'reviewer',
+        'review__proposal',
+        )
```

Comment:
This will also pull in decisions someone made as a supervisor (from time to time it happens that a committee member is also a supervisor). Thus, we'd need to filter out all supervisor decisions. Currently, we still don't have a nice way to do this (see #350). So, we need to check if the review 'stage' instead.
Adding ``review__stage__gt=Review.SUPERVISOR`` to the decision QS should work.